plugins/rapid7_intsights/plugin.spec.yaml

```
@@ -495,7 +495,7 @@ actions:
         example: ["DarkWeb"]
       matched_asset_value:
         title: Matched Asset Value
-        description: List of matched asset values, example. domain name, company name, IP
+        description: 'List of matched asset values (examples: domain name, company name, IP)'
```

```suggestion
        description: 'List of matched asset values. Examples: domain name, company name, IP'
```
Making this consistent with line 829